chaste_codegen/data/tests/all_models_reference_models/BE/Shannon2004.cpp

```diff
@@ -2029,7 +2047,7 @@ void OdeSystemInformation<CellShannon2004FromCellMLBackwardEuler>::Initialise(vo
     this->mParameterUnits.push_back("milliS_per_microF");
 
     // mParameters[11]:
-    this->mParameterNames.push_back("membrane_sodium_calcium_exchanger_current_conductance");
+    this->mParameterNames.push_back("INaCa__V_max");
```

Comment:
This change looks wrong too. Hopefully a consequence of the INPUT instead of OUTPUT.

Reply:
This is a logical consequence of the conversion: the conversion moving the metadata tag so changing to output doesn't change that.

Also: wouldn't it be weird to have a parameter and a derived quantity named the same (but in different units?)

Reply:
Ah, yes, that would be odd! I didn't realise it was both. As Gary says, we don't really want conductances to be considered derived quantities and parameters. Perhaps the logic here needs to be the same as for `GetIIonic` rather than using the ontology? Or we need to fix the ontology to support this use case properly!

Reply:
This issue should now be fixed due to the ontology fix, this seems to be the case looking at  `chaste_codegen/data/tests/all_models_reference_models/BE/Shannon2004.cpp `